src/widgets/settingspages/EmotesPage.cpp

```diff
@@ -19,6 +19,8 @@ EmotesPage::EmotesPage()
     //    layout.append(this->createCheckBox("Enable FrankerFaceZ emotes",
     //    settings.enableFfzEmotes)); layout.append(this->createCheckBox("Enable
     //    emojis", settings.enableEmojis));
+    //    layout.append(this->createCheckBox("Enable custom emotes",

```

Comment:
This is a deprecated page, use GeneralPage instead for this setting

Reply:
added commented setting for documentation

the other enableXEmotes arent on the general page, so I added it nearby them